src/app/modules/myinfo/myinfo.util.ts

```diff
@@ -83,9 +137,12 @@ const hasMyInfoAnswer = (
 
 const transformAnswer = (field: VisibleMyInfoResponse): string => {
   const answer = field.answer
-  return field.fieldType === BasicField.Date
-    ? moment(new Date(answer)).format('YYYY-MM-DD')
-    : answer
+  switch (field.fieldType) {
+    case BasicField.Date:
+      return moment(new Date(answer)).format('YYYY-MM-DD')
+    default:
+      return field.answer
+  }
```

Comment:
Why this change?

Also, default can be `return answer`

Reply:
Woops stray change. Reverting. Thank you.